docusaurus/docs/quickstarts/zac/installation.md

```diff
@@ -181,10 +181,14 @@ internal docker compose network that is declared in the compose file. You would
    3. Example using AWS "host it anywhere":
       ![host it anywhere](./zac_configure_hia.png)
 
-3. Login with admin/admin
- 
+3. Login as admin
+
+   If you followed the quickstart then admin's password is stored as an environment variable `ZITI_PWD` defined in your quickstart environment file. You may reload your quickstart environment by sourcing the environment file: `source ~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env`.
+
    ![img_2.png](./zac_login.png)
 
-4. **IMPORTANT!!!** Edit your profile and change the password
+4. Optionally, edit your profile to change admin's password
 
    ![img_3.png](./zac_change_pwd.png)
+
+   Keep in mind that this is the same password that is sourced from the quickstart environment file. You'll need to change it there too for the `zitiLogin` command.

```

Comment:
same comment from before - don't need it. also redundant for what is in step 3. maybe just combine some of this with step 3 and eliminate 4 entirely?